ext/ffi/lib.rs

```diff
@@ -349,6 +427,12 @@ impl NativeValue {
       NativeType::Pointer | NativeType::Function | NativeType::Buffer => {
         Value::from(self.pointer as usize)
       }
+      NativeType::Struct(_) => {
+        let slice = Box::from_raw(self.struct_value);
+        let (ptr, len) = out_buffer.unwrap();
+        ptr.copy_from_nonoverlapping(slice.as_ptr(), len);
```

Comment:
Why is the struct_value intermittent necessary? Couldn't it just be passed the `v8::TypedArray` as a pointer?